kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -198,10 +202,29 @@
     },
     $trNameSpace: 'userPage',
     $trs: {
-      noUsersExist: 'No Users Exist.',
-      allUsersFilteredOut: 'No users match the filter.',
+      // input & accessibility labels
```

Comment:
the main thing I don't understand is how translators are supposed to fully understand the context of what they're translating. Would be optimal if somehow they could see a screenshot of the page with the text highlighted.

Assuming they've only got the `trNameSpace` and `$tr` key to work with, seems we should make the key as descriptive as possible?

So `filter.allUsers` would be more better than just `allUsers`. Or, we could even be more verbose:

```
$trs: {
  'placeholder message - no users exist': 'No Users Exist.',
  'placeholder message - all users are filtered out': 'No users match the filter.',
  'button label - edit account': 'Edit Account Information',
  'filter selection label - filter type': 'Filter User Type',
  'filter selection option - allUsers': 'All Users',
  'filter selection option - admins': 'Admins',
```

I'm not sure what translators' expectations or needs are...